be/src/service/service_be/lake_service.cpp

```diff
@@ -478,28 +478,38 @@ void LakeServiceImpl::get_tablet_stats(::google::protobuf::RpcController* contro
                                        const ::starrocks::lake::TabletStatRequest* request,
                                        ::starrocks::lake::TabletStatResponse* response,
                                        ::google::protobuf::Closure* done) {
+    static constexpr int64_t kDefaultTimeout = 5 * 60 * 1000L; // 5 minutes
```

Comment:
can move this const to LakeServiceImpl class static member named as `kDefaultTimeoutForGetTabletStats`